.github/workflows/ci-tests.yml

```diff
@@ -207,6 +207,24 @@ jobs:
       shell: bash
       run: |
         ./scripts/run-test-codegen-configurations.sh -t
+  
+  verify-frontend-bundle-latest:
```

Comment:
Would it be more efficient to include this as a step in the codegen unit tests CI job instead of spinning up another job? Not sure if it matters since the CI jobs on these runners are free minutes?